pkgs/development/python-modules/trio/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder
+, attrs
+, sortedcontainers
+, async_generator
+, idna
+, outcome
+, contextvars
+, pytest
+, pyopenssl
+, trustme
+}:
+
+buildPythonPackage rec {
+  pname = "trio";
+  version = "0.4.0";
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0ib1x47knlad9pljb64ywfiv6m3dfrqqjwka6j1b73hixmszb5h4";
+  };
+
+  checkInputs = [ pytest pyopenssl trustme ];
```

Comment:
You're sure the last two are only needed for testing?

Reply:
Yes, they're only listed in test-requirements.txt, not in setup.py; and I don't believe there's any place that the trio library would use them.